test/factories/host_related.rb

```diff
@@ -320,6 +320,24 @@ def set_environment_taxonomies(host_or_hostgroup, environment = host_or_hostgrou
       end
     end
 
+    factory :host_for_snapshots_ipv4_dhcp_el7 do
```

Comment:
Why a separate factory? Shouldn't it be a [trait](https://github.com/thoughtbot/factory_bot/blob/master/GETTING_STARTED.md#defining-traits)?

Reply:
That does not work either, I have no clue why but it looks like for traits some orchestration is being triggered.